src/hooks/useRouter/index.js

```diff
@@ -0,0 +1,38 @@
+import { useMemo } from 'react'
```

Comment:
why are these hooks in an index.js inside a directory instead of just in a single file?